homework lession 4.py

```diff
@@ -0,0 +1,39 @@
+
+
+#1. Створити програму, яка буде очікувати введення тексту від користувача і повідомляти — чи є введений текст “числом” чи “словом” (використати функцію cтрічки .isdigit() або .isalpha())
+
+month = input("What month is today?")
+if(month.isdigit()):
```

Comment:
nit: add space after `if` and brackets should be ommited
```suggestion
if month.isdigit():
```